src/components/LogoutListener/helpers/useLogoutFromMultipleTabs.ts

```diff
@@ -2,12 +2,15 @@ import { useEffect } from 'react';
 import { useGetAccountInfo } from 'hooks';
 import { logout } from 'utils/logout';
 import { localStorageKeys } from 'utils/storage/local';
+import { useSelector } from 'reduxStore/DappProviderContext';
+import { autoLogoutCallbackUrlSelector } from 'reduxStore/selectors';
```

Comment:
rename `autoLogoutCallbackUrl` to `logoutRoute` (since route will be auto-transformed in url only for wallet)

Reply:
done